hypothesis-python/tests/common/debug.py

```diff
@@ -18,7 +18,7 @@
 
 from tests.common.utils import no_shrink
 
-TIME_INCREMENT = 0.01
+TIME_INCREMENT = 0.001
```

Comment:
`_consistently_increment_time` was giving me trouble with spurious deadline errors, so I've just bumped our increment down so we don't hit 200ms. The misalignment logic hits more time.time() calls than before via constructing ConjectureDatas (`self.start_time = time.perf_counter()`).